server/routes/getRewards.ts

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable guard-for-in */
-import Sequelize from 'sequelize';
+// import Sequelize from 'sequelize';
 import BN from 'bn.js';
 // import _ from 'lodash';
 import moment from 'moment';
 import { Request, Response, NextFunction } from 'express';
 import { factory, formatFilename } from '../../shared/logging';
 import { Errors } from './getOffences';
+const Sequelize = require('sequelize');
+import {sequelize} from './../database'
```

Comment:
spacing shouldl have spaces before and after brackets

Reply:
fixed in latest commit